app/src/main/java/com/gatheringhallstudios/mhworlddatabase/data/dao/SkillDao.kt

```diff
@@ -21,7 +21,7 @@ abstract class SkillDao {
         SELECT id, name, description, icon_color
         FROM skilltree s join skilltree_text st USING (id)
         WHERE lang_id = :langId
-        ORDER BY id ASC""")
+        ORDER BY icon_color DESC, name """)
```

Comment:
Why would you sort by icon color? By name is fine though. Asked in the slack about the change and it might be a good call.